chatgpt/nodes/validator.py

```diff
@@ -148,16 +179,25 @@ def score_validator_performance(self, own_submission: Dict[str, Any], validator_
 
         scores = {}
 
-        for dimension in ['overall_score', 'authenticity', 'ownership', 'quality', 'uniqueness']:
+        for dimension in ['score', 'authenticity', 'ownership', 'quality', 'uniqueness']:
             if dimension in own_submission and dimension in validator_score:
-                scores[dimension] = 1 - abs(own_submission[dimension] - validator_score[dimension]) / 100
+                scores[dimension] = 1 - abs(own_submission[dimension] - validator_score[dimension] / 1e18)
```

Comment:
IMO we should apply the `/ 1e18` normalization step when `validator_score` is defined.